src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -315,13 +316,20 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
     }
   ];
 
+  const statusTypes = reimbursementRequest.reimbursementStatuses.map((status) => status.type);
```

Comment:
we should only display the most recent status, so sort the statues by dateCreated and use the most recent one